### DISCUSSION TITLE
How to style checkbox input

### DISCUSSION THREAD

**Discussion by gerrykeys** [microsoft/griffel#527] [Q&A] [answered]
I'm using React Fluent UI 2 Library and i want to style checkbox component using makeStyles():
I can't understand how to build the object for the makeStyles function in order to select `<input type="checkbox">`  element for change its background and checkmark color.
Can provide me some suggestions about?
Thanks

**Comment by gerrykeys** [comment]
I Solved the issue.
Fluent provides slots which is possible to modify using griffel.

**Reply by spmonahan** [reply, marked as answer]
Here's the relevant documentation for reference: [Customizing components with slots](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page)
